llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -1108,19 +1118,19 @@ void llvm::ComputeCrossModuleImport(
           for (const auto &VI : GVS->refs()) {
             // Try to emplace the declaration entry. If a definition entry
```

Comment:
nit: can remove the comment "// Try to emplace the declaration entry ..."?

Same below at lines 1128 and 1133